cmake/sysbuild/partition_manager.cmake

```diff
@@ -0,0 +1,817 @@
+  else()
+    set(user_def_pm_static ${PM_STATIC_YML_FILE})
+    set(image_name ${DOMAIN_APP_APP})
+  endif()
+
+  sysbuild_get(${image_name}_APPLICATION_CONFIG_DIR IMAGE ${image_name} VAR APPLICATION_CONFIG_DIR CACHE)
+
+  ncs_file(CONF_FILES ${${image_name}_APPLICATION_CONFIG_DIR}
+           PM conf_dir_pm_static
+           DOMAIN ${DOMAIN}
+           BUILD ${CONF_FILE_BUILD_TYPE}
+  )
+
+  ncs_file(CONF_FILES ${BOARD_DIR}
+           PM board_dir_pm_static
+           DOMAIN ${DOMAIN}
+           BUILD ${CONF_FILE_BUILD_TYPE}
+  )
+
+  if(EXISTS "${user_def_pm_static}" AND NOT IS_DIRECTORY "${user_def_pm_static}")
+    set(static_configuration_file ${user_def_pm_static})
+  elseif (EXISTS ${conf_dir_pm_static})
+    set(static_configuration_file ${conf_dir_pm_static})
+  elseif (EXISTS ${board_dir_pm_static})
+    set(static_configuration_file ${board_dir_pm_static})
+  endif()
+
+  if (EXISTS ${static_configuration_file})
+    message(STATUS "Found partition manager static configuration ${PM_DOMAIN}: "
+                   "${static_configuration_file}"
+    )
+    set(static_configuration --static-config ${static_configuration_file})
+  endif()
+
+  if (NOT static_configuration AND CONFIG_PM_IMAGE_NOT_BUILT_FROM_SOURCE)
+    message(WARNING
+      "One or more child image is not configured to be built from source. \
+      However, there is no static configuration provided to the \
+      partition manager. Please provide a static configuration as described in \
+      the 'Scripts -> Partition Manager -> Static configuration' chapter in the \
+      documentation. Without this information, the build system is not able to \
+      place the image correctly in flash.")
+  endif()
+
+
+# We must set this when running for the domain, so how is the domain name partition handled in settings ?
+  if("${PM_DOMAIN}" STREQUAL "CPUNET")
+    set(dynamic_partition ${${PM_DOMAIN}_PM_DOMAIN_DYNAMIC_PARTITION})
+    set(
+      dynamic_partition_argument
+      "--flash_primary-dynamic-partition;${dynamic_partition}"
+      )
+  endif()
+
+  if (DEFINED PM_DOMAIN)
+    set(underscore _)
+  else()
+    set(underscore)
+  endif()
+
+  set(pm_out_partition_file ${APPLICATION_BINARY_DIR}/partitions${underscore}${PM_DOMAIN}.yml)
+  set(pm_out_region_file ${APPLICATION_BINARY_DIR}/regions${underscore}${PM_DOMAIN}.yml)
+  set(pm_out_dotconf_file ${APPLICATION_BINARY_DIR}/pm${underscore}${PM_DOMAIN}.config)
+
+  set(pm_cmd
+    ${PYTHON_EXECUTABLE}
+    ${ZEPHYR_NRF_MODULE_DIR}/scripts/partition_manager.py
+    --input-files ${PM_IN_FILES}
+    --regions ${PM_REGIONS}
+    --output-partitions ${pm_out_partition_file}
+    --output-regions ${pm_out_region_file}
+    ${dynamic_partition_argument}
+    ${static_configuration}
+    ${${PM_DOMAIN}${underscore}region_arguments}     # region args are scoped in and thus available. Should probably be in arg.
+    )
+  # ToDo: add_region is done only for main app, how about DOMAIN ?
```

Comment:
Comment need addressing?

Reply:
left over comment, this is handled.
Comment removed 👍